README.md

```diff
@@ -1,70 +1,34 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
```

Comment:
- Kindly the getting started header and add some instruction on how users can have your project up and running locally in their machine.